packages/nx-plugin/src/executors/prisma/executor.ts

```diff
@@ -9,16 +9,33 @@ export default async function runExecutor({
   // The options are for executors are parsed with yargs, so arguments without a `-` or `--` are added to the `_` property.
   // We assume these are all prisma commands like `prisma migrate reset`.
   // Then we add the schema argument and any remaining standard arguments.
-  const command = `npx prisma ${_.join(
-    ' '
-  )} --schema=${schema} ${Object.entries(options)
-    .map(([key, value]) => `--${key}=${value}`)
-    .join(' ')}`;
 
   // `inherit` allows the parent terminal to send and receive input and output on stdin/out/err.
   // This is necessary because prisma commands are interactive.
-  execSync(command, {
-    stdio: 'inherit',
+  const child = spawn(
+    'npx',
+    [
+      'prisma',
+      ..._,
+      `--schema=${schema}`,
+      ...Object.entries(options).map(([key, value]) => `--${key}=${value}`),
+    ],
+    {
+      stdio: ['inherit', 'pipe', 'inherit'],
```

Comment:
Can you try adding `shell: true` to these options and report what happens?

Reply:
That seemed to fix it for me!